furms-core/src/main/java/io/imunity/furms/core/projects/ProjectServiceImpl.java

```diff
@@ -248,6 +249,15 @@ public boolean isAdmin(String projectId){
 		return authzService.isResourceMember(projectId, PROJECT_ADMIN);
 	}
 
+	@Override
+	@FurmsAuthorize(capability = PROJECT_READ, resourceType = PROJECT, id = "projectId")
+	public boolean hasAdminRights(String projectId) {
```

Comment:
The term admin rights is not precisely defined. 
Please be more direct: isProjectOrCommunityAdmin.
Note that community admin needs not to be a superset of project admin.